android-svc-lib.sh

```diff
@@ -149,12 +162,22 @@ CallServiceMethod () {
     ParseParcel "${l_parcel}" "${l_returnDataType}"
 }
 
+###
+# list available service package names from connected android devices
```

Comment:
`from connected android devices` - It only handles one device at a time and it doesn't have to be a "connected" device, as you can run it directly on the device without adb.